### PR TITLE
[next] chore(master): merge `master` into `next`

### PR DESCRIPTION
**Pull request by raimund-schluessler**
### ☑️ Resolves

* Merges `master` into `next`.

This was one hell of a backport :see_no_evil: But I think I got everything working. And it also fixes the search box of `NcEmojiPicker` that currently does not work for `next`.

After this is merged, I want an alpha release of `nextcloud/vue@9.0.0-alpha.0` :wink:

### CODE REVIEW

**Review by susnux — APPROVED** [review]

**Comment by Antreesy** [comment]
Ah, I had a prepared backport at my laptop for you 🙈

File: cypress/component/richtext.cy.ts [hunk 1]
```diff
@@ -515,6 +568,49 @@ describe('NcRichText', () => {
...
+						text: testCases.map(i => i.input).join('\n'),
+						useExtendedMarkdown: true,
+					},
+				})
+
+				cy.get('ul').should('exist')
+				cy.get('li').should('have.length', testCases.length)
+				cy.get('li').each((item, index) => {
+					// Vue 2.7 renders three non-breaking spaces here for some reason
+					expect(item).have.text('   ' + testCases[index].output)
```
Comment (Antreesy) [inline cypress/component/richtext.cy.ts:590]:
```suggestion
					expect(item).have.text(testCases[index].output)
```
This test fails for me at least locally

**Review by Antreesy — APPROVED** [review]

File: cypress/component/richtext.cy.ts (hunk 1 above)
Comment (ShGKme) [reply cypress/component/richtext.cy.ts:590]: Probably, should be `contains`

**Review by ShGKme — APPROVED** [review]

File: cypress/component/richtext.cy.ts (hunk 1 above)
Comment (raimund-schluessler) [reply cypress/component/richtext.cy.ts:590]: Weird, the test seems to run fine on CI, no?

Comment (Antreesy) [reply cypress/component/richtext.cy.ts:590]: Seems to run fine, as we see it. But I believe, Grigorii's proposal should fit here better

Comment (susnux) [reply cypress/component/richtext.cy.ts:590]: I also have this issue on master locally.

**Pull request merged**